builds/ngap/beslog2json.py

```diff
@@ -0,0 +1,560 @@
+    debug(f"TRANSMIT_INFO_LOG is {eord(TRANSMIT_INFO_LOG)}")
+    debug(f"TRANSMIT_ERROR_LOG is {eord(TRANSMIT_ERROR_LOG)}")
+    debug(f"TRANSMIT_VERBOSE_LOG is {eord(TRANSMIT_VERBOSE_LOG)}")
+    debug(f"TRANSMIT_TIMING_LOG is {eord(TRANSMIT_TIMING_LOG)}")
+
+
+##########################################################
+# Default configuration for what gets written out as JSON
+# These may be overridden by command line options.
+TRANSMIT_REQUEST_LOG = True
+TRANSMIT_INFO_LOG    = True
+TRANSMIT_ERROR_LOG   = True
+TRANSMIT_VERBOSE_LOG = True
+TRANSMIT_TIMING_LOG  = False
+
+##########################################################
+# Keys Common To All Log Formats
+TIME_KEY="time"
+PID_KEY="pid"
+TYPE_KEY="type"
+
+##########################################################
+# Log Message Type Keys
+REQUEST_MESSAGE_TYPE="request"
+INFO_MESSAGE_TYPE="info"
+ERROR_MESSAGE_TYPE="error"
+VERBOSE_MESSAGE_TYPE="verbose"
+TIMING_MESSAGE_TYPE="timing"
+
+##########################################################
+# Timing Log Keys
+MESSAGE_KEY="message"
+ELAPSED_TIME_KEY="elapsed_us"
+START_TIME_KEY="start_us"
+STOP_TIME_KEY="stop_us"
+REQUEST_ID_TIMER_KEY="request_id"
+TIMER_NAME_KEY="timer_name"
+
+######################################################################################################
+# Request Log Keys
+# Log Fields type==request
+# 1601646465|&|2122|&|request|&|OLFS|&|0:0:0:0:0:0:0:1|&|USER_AGENT|&|92F3C71F959B56515C98A09088CA2A8E
+#    0          1        2       3           4              5               6
+# |&|-|&|1601646465304|&|REQUEST-ID|&|HTTP-GET|&|/opendap/hyrax/data/nc/fnoc1.nc.dds|&|u|&|BES
+#    7     8              9             10              11                             12   13
+# |&|get.dds|&|dap2|&|/Users/ndp/OPeNDAP/hyrax/build/share/hyrax/data/nc/fnoc1.nc|&|u
+#      14       15                              16                                  17
+#
+#  0: start_time
+#  1: pid
+#  2: request type
+#  3: olfs boundary tag (ignored)
+#  4: client_ip
+#  5: user_agent
+#  6: session_id
+#  7: user_id
+#  8: olfs_start_time
+#  9: request_id
+# 10: http_verb
+# 11: url_path
+# 12: query_string
+# 13: bes boundary key (ignored)
+# 14: bes action
+# 15: return as
+# 16: local_path
+# 17: ce
+
+CLIENT_IP_KEY="client-ip" # field 4
+USER_AGENT_KEY="user-agent" # field 5
+SESSION_ID_KEY="session-id" # field 6
+USER_ID_KEY="user-id" # field 7
+OLFS_START_TIME_KEY="olfs-start-time" # field 8
+REQUEST_ID_KEY="requiest-id" # field 9
+HTTP_VERB_KEY="http-verb" # field 10
+URL_PATH_KEY="url-path" # field 11
+QUERY_STRING_KEY="query-string" # field 12
+BES_ACTION_KEY="bes-action" # field 14
+RETURN_AS_KEY="return-as" # field 15
+LOCAL_PATH_KEY="local-path" # field 16 
+CE_KEY="constraint-expression" # field 17
+
+
+##########################################################
+# request_log_to_json()
+# Ingest a BES request log record.
+#
+def request_log_to_json(log_fields, json_log_line):
```

Comment:
looks like `send_it=False` by default within the scope of `request_log_to_json`. How about:
```python
def request_log_to_json(log_fields, json_log_line, send_it=False):
```